packages/react-kit/tsconfig.json

```diff
@@ -1,8 +1,6 @@
 {
   "extends": "./node_modules/@devexperts/tools/settings/tsconfig.json",
   "compilerOptions": {
-	"noUnusedLocals": false,
-	"strictNullChecks": true,
```

Comment:
removed this because base tsconfig in tools package already has strictNullChecks option.